libcipher/transpo.py

```diff
@@ -0,0 +1,49 @@
+"""Transposition cipher implementation
+
+Provides functions to encrypt / decrypt a message by changing the
+character position of plain text.
+
+>>> decrypt(8, 'Cenoonommstmme oo snnio. s s c')
+'Common sense is not so common.'
+
+"""
+from __future__ import division
+import math
+
+
+def decrypt(key, message):
+    """Returns message decrypted using Transposition Cipher
+
+    The Transposition decrypt function will simulate the columns and
+    rows of the grid that the plain text is written on by using a list of
+    strings. No of Columns is calculated by (length of the text)/key.
+    key value is consider as no of rows.
+
+    Args:
+         key (int): Encryption key to decrypt message
+         message (str): Encrypted message
+
+    Returns:
+         message (str): Decrypted message.
```

Comment:
The `message` is redundant here. Please check the Google docstring convention for this.